docs/source/tutorial/tutorial-pagination.md

```diff
@@ -142,20 +144,10 @@ Next, you need to add some handling when the cell is tapped. Normally that's han
 
 Luckily, you can override the `shouldPerformSegue(withIdentifier:sender:)` method to say, "In this case, don't perform this segue, and take these other actions instead."
 
-Override this method, and add code that performs the segue for anything in the `.launches` section and _doesn't_ perform it (instead loading more launches if needed) for the `.loading` section:
+Update the code in this method to perform the segue for anything in the `.launches` section and _not_ perform it (instead loading more launches if needed) for the `.loading` section. Replace the `TODO` and everything below it with: 
```

Comment:
It's unclear whether this code needs to be added to `prepare(for segue:)` or `shouldPerformSegue(withIdentifier:sender:)`. The fact that it returns hints towards `shouldPerformSegue` but there appears to be a missing piece of code there to declare `listSection` used in the `switch` statement.